tests/test_coreg/test_affine.py

```diff
@@ -268,34 +372,12 @@ def test_nuth_kaab(self) -> None:
 
         # Check that the x shift is close to the pixel_shift * image resolution
         assert all(
-            abs((transformed_points.geometry.x.values - self.points.geometry.x.values) - pixel_shift * self.ref.res[0])
+            abs((transformed_points.geometry.x.values - self.points.geometry.x.values) + pixel_shift * self.ref.res[0])
```

Comment:
was it a mistake in the older test?

Reply:
That or the missing sign you noticed above, I will check.